### ISSUE TITLE
Skip rollup not working?

### ISSUE THREAD

**Issue by domagojk** [jspm/jspm-cli#1569]
I am testing out JSPM 0.17 and wanted to disable rollup
I found that option is "---skip-rollup"

```
jspm build main.js - x [outfile] Create an optimized static single-file build
  --skip-rollup                  Skip Rollup static optimizations
  --minify --no-mangle --config <build.json>
  --format <amd|cjs|umd|esm>     Specify a custom output format for the build
  --format global --global-name <g> --global-deps "{globalModuleMap}"
```

but after using it, I still get error "while using Rollup"

```
MacBook-2:web domagoj$ jspm build src/start.js test.js --skip-rollup
     Creating the single-file build for src/start.js...

err  Error: Duplicate dependency "./config/config" while using Rollup on "local:ca6-core@1.0.2/core.js". Rename the dependency, or post to https://github.com/rollup/rollup/issues/424.
```

**Comment by guybedford** [comment]
Thanks, fixed in https://github.com/jspm/jspm-cli/commit/6cfc89cbf3bbb69ba001f1c0f274e50895e5daf1.

**Comment by guybedford** [comment]
Released in 0.17.0-beta.10.
